ompi/mca/coll/ftagree/coll_ftagree_earlyreturning.c

```diff
@@ -3331,41 +3331,3 @@ int mca_coll_ftagree_iera_intra(void *contrib,
 
     return OMPI_SUCCESS;
 }
-
```

Comment:
I would not remove other people's code without prior discussion. You feel strong about it, comment or #ifdef it out, but don't remove it.

Reply:
That's why I asked for your review.  😄   This is dead code -- is it ok to remove?  I couldn't find it used anywhere.

Reply:
@bosilca and I had a lengthy conversation about this code this afternoon.  He's pretty sure that this code *should* be being called from somewhere; otherwise, we likely have a memory leak in MPI_FINALIZE.  So for this PR, I've `#if 0`'ed out this code with an appropriate comment.  @bosilca's team is looking into this (outside the scope of this PR).